eslint-plugin/lib/standalone.js

```diff
@@ -22,21 +22,35 @@
  */
 "use strict";
 
-const rules = require("./rule-list");
+const rulesList = require("./rule-list");
 
-const ruleModules = rules.reduce((map, rule) => {
-  map[rule.ruleName] = rule.ruleModule;
-  return map;
-}, {});
+const allRules = {};
+const recommendedRules = {}; // recommended rules
 
-const ruleConfigs = rules.reduce((map, rule) => {
-  const recommended = rule.ruleModule.meta.docs.recommended;
-  map[`@ecocode/${rule.ruleName}`] =
-    recommended === false ? "off" : recommended;
-  return map;
-}, {});
+for (let { ruleName, ruleModule } of rulesList) {
+  allRules[ruleName] = ruleModule;
+  const { recommended } = ruleModule.meta.docs;
+  const ruleConfiguration = recommended === false ? "off" : recommended;
+  recommendedRules[`@ecocode/${ruleName}`] = ruleConfiguration;
+}
 
-module.exports = {
-  rules: ruleModules,
-  configs: { recommended: { plugins: ["@ecocode"], rules: ruleConfigs } },
+const plugin = {
+  meta: {
+    name: "@ecocode/eslint-plugin",
+    version: "1.6.0",
```

Comment:
Agree not fan neither

The ESLint configuration says:

> you should add a meta key that contains **at least a name key, and ideally, a version key**

So version is not mandatory and looking a other plugins, it looks like they don't include it, so we could drop it.

I'll just do some check because it also says:

> Without this meta information, your plugin will not be usable with the --cache and --print-config command line options.

Even if the version is not mandatory, I would like to make sure about its impact regarding the cache feature